index.html

```diff
@@ -32,13 +34,124 @@ <h5>
+              <li>javaScript</li>
+            </ul>
+      <button class="btn_details">See Project</button>
+    </div>
+    <div class="card">
+      <img src="./pictures/Snapshoot-Portfolio1.png" alt="Project_Image" srcset="">
+      <h2>Multi-Post <br>
+      Stories</h2>
+      <div class="title">
+        <h5>CANOPY</h5>
+        <ul class="title_list">
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+      </div>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="tech_list">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+      <button class="btn_details">See Project</button>
+    </div>
+    <div class="card">
+      <img src="./pictures/Snapshoot-Portfolio2.png" alt="Project_Image" srcset="">
+      <h2>Tonic</h2>
+      <div class="title">
+        <h5>CANOPY</h5>
+        <ul class="title_list">
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+      </div>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="tech_list">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+      <button class="btn_details">See Project</button>
+    </div>
+    <div class="card">
+      <img src="./pictures/Snapshoot-Portfolio3.png" alt="Project_Image" srcset="">
+      <h2>Multi-Post<br>
+      Stories</h2>
+      <div class="title">
+        <h5>CANOPY</h5>
+        <ul class="title_list">
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>
+      </div>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="tech_list">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+      <button class="btn_details">See Project</button>
+    </div>
+  </section>
+  <section class="about_section">
+        <h2>About Myself</h2>
+        <p>
+        Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+        experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <h5>
+          LET’S CONNECT
+        </h5>
+    <ul>
+        <li><a href="#"><i class="fa-brands fa-github socialMedia"></i></a></li>
+        <li><a href="#"><i class="fa-brands fa-linkedin-in socialMedia"></i></a> </li>
+        <li><a href="#"><i class="fa-regular fa-hand-peace socialMedia"></i></a></li>
+        <li><a href="#"><i class="fa-brands fa-twitter socialMedia"></i></a></li>
+        <li><a href="#"><i class="fa-solid fa-m socialMedia"></i></a></li>
+    </ul>
+    <button class="btn_details">Get my resume</button>
+    <ul class="drop_down">
+      <li class="languages_title"><h3>Languages</h3>
```

Comment:
- [OPTIONAL] The headers for the dropdown have a `font-size` of `20px.` That is the _**Language**_, _**Framework**_ and the _**Skills**_ headers. It would be nice to implement this change as it would make your design identical to the one you chose. 👍